### PR TITLE
task-1 added

### PR DESCRIPTION
**Pull request by lenadgit**
_Complete the form below and remove this line._

### task name

[Demo](link-to-webpage-on-github-pages) |
[Code base](link-to-complete-codebase-in-different-repo)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by lenadgit** [comment]
task-1 Popup

File: submissions/lenadgit/task-1/css/style.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+.content {
+	width: 320px;
+	height: auto;
+	right: 60px;
+	padding-top: 40px;
+}
+
+.inner-popup {
+	border: 1px solid #ccc;
```
Comment (A-Ostrovnyy) [inline submissions/lenadgit/task-1/css/style.css]: Use Prettier for formatting your code.

File: submissions/lenadgit/task-1/css/style.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+	float: left;
+	padding: 0 20px 20px 20px;
+	border: 1px solid #fff;
+}
+
+.icon:hover {
+	border: 1px solid #ccc;
+}
+
+
```
Comment (A-Ostrovnyy) [inline submissions/lenadgit/task-1/css/style.css]: Remove unused lines.

File: submissions/lenadgit/task-1/css/style.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+.btn:active {
+	color: transparent;
+}
+
+.inner-pics:hover {
+	display: block; 
+	z-index: 3;
+	margin-top: -30px;
+	background: #fff;
+}
```
Comment (A-Ostrovnyy) [inline submissions/lenadgit/task-1/css/style.css]: Add an empty line at the end of the file.

File: submissions/lenadgit/task-1/index.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  <body>
+<div class = "container">
+  <div class = "menu">
+<div class = "tabs">Gmail</div>
+<div class = "tabs">Images</div>
+<div class="tabs">
+  <div class="content">
+       <div class = "inner-popup">
+      <div class = "icons">
+        <!--<ul>
```
Comment (A-Ostrovnyy) [inline submissions/lenadgit/task-1/index.html]: Do not commit commented lines of code.

File: submissions/lenadgit/task-1/index.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+          <li class = "icon"><a href = ""><img src = "img/doc.jpg"></a></li>
+          <li class = "icon"><a href = ""><img src = "img/translate.jpg"></a></li>
+          <li class = "icon"><a href = ""><img src = "img/google.jpg"></a></li>
+          <li class = "icon"><a href = ""><img src = ""></a></li>
+          <li class = "icon"><a href = ""><img src = ""></a></li>
+        </ul>  -->
+
+        <div class = "icon"><a href = ""><img src = "img/doc.jpg"></a></div>
```
Comment (A-Ostrovnyy) [inline submissions/lenadgit/task-1/index.html]: Don't forget about `alt` attribute of `` tag.

File: submissions/lenadgit/task-1/index.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        <div class = "icon"><a href = ""><img src = "img/mail.jpg"></a></div>  
+        <div class = "icon"><a href = ""><img src = "img/disk.jpg"></a></div>
+        <div class = "icon"><a href = ""><img src = "img/calendar.jpg"></a></div>
+        <div class = "clearfix"></div> 
+        <div class = "icon"><a href = ""><img src = "img/business.jpg"></a></div> 
+        <div class = "clearfix"></div>
+      </div>
+      <button class = "btn">More
+<div class ="inner-pics">        <div class = "icon"><a href = ""><img src = "img/doc.jpg"></a></div> 
+        <div class = "icon"><a href = ""><img src = "img/translate.jpg"></a></div>
```
Comment (A-Ostrovnyy) [inline submissions/lenadgit/task-1/index.html]: Good practice use `href=#` for empty links.

**Pull request merged**